tests/flow/Install_RedisGears.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+ROOT=$(cd $HERE/../..; pwd)
+READIES=$ROOT/opt/readies
+. $READIES/shibumi/defs
+
+if [[ $1 == --help || $1 == help || $HELP == 1 ]]; then
+	cat <<-END
+		Obtain RedisGears module binaries
+
+		Install_RedisGears.sh [--help|help]
+		
+		Argument variables:
+		GEARS_OSNICK=nick   Get binaries for give osnick
+		GEARS_PATH=dir      Get binaries from given Gears repo
+		NOP=1               No operation
+		HELP=1              Show help
+
+	END
+	exit 0
+fi
+
+OP=""
+[[ $NOP == 1 ]] && OP=echo
+
+os=$($READIES/bin/platform --os)
+arch=$($READIES/bin/platform --arch)
+
+if [[ ! -z $GEARS_PATH ]]; then
+	platform=$($READIES/bin/platform -t)
+else
+	if [[ $os != linux || $arch != x64 ]]; then
+		eprint "Cannot match binary artifacts - build RedisGears and set GEARS_PATH"
+		exit 1
+	fi
+
+	dist=$($READIES/bin/platform --dist)
+	nick=$($READIES/bin/platform --osnick)
+
+	if [[ $dist == ubuntu ]]; then
+		if [[ $nick != bionic && $nick != xenial && $nick != trusty ]]; then
+			nick=bionic
+		fi
+	elif [[ $dist == debian ]]; then
+		nick=bionic
+	elif [[ $dist == centos || $dist == redhat || $dist == fedora ]]; then
+		nick=centos7
+	elif [[ ! -z $GEARS_OSNICK ]]; then
+		nick="$GEARS_OSNICK"
+	else
+		eprint "Cannot match binary artifacts - build RedisGears and set GEARS_PATH"
+		exit 1
+	fi
+	platform="${os}-${nick}-${arch}"
+fi
+
+GEARS_S3_URL=http://redismodules.s3.amazonaws.com/redisgears/snapshots
+GEARS_MOD=redisgears.$platform.master.zip
+GEARS_DEPS=redisgears-python.$platform.master.tgz
+
+FINAL_WORK_DIR=$ROOT/bin/$($READIES/bin/platform -t)/RedisGears
```

Comment:
Since line 31 has a call to platform -t, what about putting all of those together in one place, then you can refer to the variable here.